event.c

```diff
@@ -724,10 +725,13 @@ event_handle_maprequest(xcb_map_request_event_t *ev)
     if(wa_r->override_redirect)
         goto bailout;
 
-    if(xembed_getbywin(&globalconf.embedded, ev->window))
+    if((em = xembed_getbywin(&globalconf.embedded, ev->window)))
     {
         xcb_map_window(globalconf.connection, ev->window);
         xembed_window_activate(globalconf.connection, ev->window);
+        /* This is not the correct way to set this, but let's be gentle... */
+        em->info.flags |= XEMBED_MAPPED;
```

Comment:
Actually set an `_XEMBED_INFO` property on the tray icon, like the xembed specification say (which Qt/Skype does not set at all). Neither of the xembed spec nor the systray spec mention anything about MapWindow.